sql/sql_class.h

```diff
@@ -722,6 +722,16 @@ typedef struct system_status_var
   ulong max_statement_time_set;
   ulong max_statement_time_set_failed;
 
+  /** number of contiguous page reads */
```

Comment:
an InnoDB data scan comment is too specific for server code in sql_class, should be something like "number of contiguous InnoDB page reads inside a query" or similar, likewise below

Reply:
fixed